src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -994,66 +1007,84 @@ $dark-focusColor: var(--green-500, color(green-500));
   align-self: center;
   // Sets the minimum width to 12 characters which is the length of the string “1234 results” which should be reasonably safe in order to avoid having layout elements jump around.
   min-width: 12ch;
-  padding-right: var(--spacing-sm, spacing(sm));
+  padding-right: var(--spacing-sm, var(--kui-space-50, $kui-space-50));
   text-align: right;
 }
 
 .k-code-block-search-results:not(.k-code-block-search-results-has-query) {
-  color: var(--grey-500, color(grey-500));
+  color: var(--grey-500, var(--kui-color-text-neutral, $kui-color-text-neutral));
 }
 
 .theme-dark .k-code-block-search-results:not(.k-code-block-search-results-has-query) {
-  color: var(--steel-300, color(steel-300));
+  color: var(--steel-300, var(--kui-color-text-neutral-weak, $kui-color-text-neutral-weak));
 }
 
 .k-code-block-search-error,
 .k-code-block-search-results {
-  margin-bottom: 0;
-  margin-top: 0;
+  margin-bottom: var(--kui-space-0, $kui-space-0);
+  margin-top: var(--kui-space-0, $kui-space-0);
 }
 
 .k-code-block-search-error {
-  background-color: var(--white, color(white));
-  border: 1px solid currentColor;
-  border-bottom-left-radius: 3px;
-  border-bottom-right-radius: 3px;
-  color: var(--red-700, color(red-700));
-  font-size: 13px;
+  background-color: var(--white, var(--kui-color-background, $kui-color-background));
+  border: var(--kui-border-width-10, $kui-border-width-10) solid currentColor;
+  border-bottom-left-radius: var(--kui-border-radius-10, $kui-border-radius-10);
+  border-bottom-right-radius: var(--kui-border-radius-10, $kui-border-radius-10);
+  color: var(--red-700, var(--kui-color-text-danger, $kui-color-text-danger));
+  font-size: var(--kui-font-size-20, $kui-font-size-20);
   left: -1px;
```

Comment:
**Suggestion**: Ensures the negative positioning matches what it means to offset: the `border` property above.

```suggestion
  left: calc(-1 * var(--kui-border-width-10, $kui-border-width-10));
```